server/src/main/java/org/elasticsearch/transport/RemoteClusterService.java

```diff
@@ -398,4 +400,16 @@ public void onFailure(Exception e) {
             });
         }
     }
+
+    /**
+     * Returns a client to the remote cluster if the given cluster alias exists.
```

Comment:
while at it, shall we mention that it throws if cluster alias is not found (rather than for instance returning null)